app/views/landing-pages/parents/PageParentsJumbotron.vue

```diff
@@ -121,4 +121,8 @@ export default {
     }
   }
 
+  #parent-page h1, #parent-page h4 {
+    font-weight: 700 !important;
```

Comment:
From the scoped style, you no longer need to reference the id `#parent-page`, as it doesn't appear in the html template.

You could scope it within the `.top-jumbotron` class? I.e. `.top-jumbotron h1`. You can remove the `h4` selector as there isn't an h4 in this vue component.

Finally we don't need the `!important`.

Reply:
Updated accordingly

Reply:
Updated css changes in `PageParentsJumbotron.vue`